### DISCUSSION TITLE
Issue with renaming the organisation

### DISCUSSION THREAD

**Discussion by nishi-sharma-db** [community/community#102742] [New to GitHub]
i am not able to see the prompt which opens after clicking on rename org. Also the GH gets hanged after clicking on rename org. Kindly help with the issue.

**Comment by queenofcorgis** [comment]
Hi @nishi-sharma-db, 

Thanks for posting - would you mind sharing a bit more about what you're experiencing so we can better assist? Any screenshots you have would be fantastic!
